pkg/addonfactory/helm_agentaddon.go

```diff
@@ -260,3 +279,47 @@ func (a *HelmAgentAddon) releaseOptions(
 		Namespace: a.getValueAgentInstallNamespace(addon),
 	}
 }
+
+// Manifest represents a manifest file, which has a name and some content.
+type Manifest struct {
```

Comment:
nit, should be private struct. I think you only use it internally?

Reply:
Done!